src/pyOpenMS/CMakeLists.txt

```diff
@@ -479,15 +302,14 @@ endif()
 #------------------------------------------------------------------------------
 # collection of include dirs that is necessary to compile pyOpenMS
 
-# Find Qt5 includes for pyOpenMS
-find_package(Qt5 COMPONENTS Core Network REQUIRED)
-get_target_property(QT_QMAKE_EXECUTABLE Qt5::qmake IMPORTED_LOCATION)
-exec_program(${QT_QMAKE_EXECUTABLE} ARGS "-query QT_INSTALL_HEADERS" OUTPUT_VARIABLE QTHEADERS)
+# Find Qt6 includes for pyOpenMS
+find_package(Qt6 COMPONENTS Core Network REQUIRED)
+get_target_property(QT_QMAKE_EXECUTABLE Qt6::qmake IMPORTED_LOCATION)
+execute_process(COMMAND ${QT_QMAKE_EXECUTABLE} -query QT_INSTALL_HEADERS  OUTPUT_VARIABLE QTHEADERS OUTPUT_STRIP_TRAILING_WHITESPACE)
 set(QT_INCLUDE_DIR ${QTHEADERS} CACHE INTERNAL "" FORCE)
-exec_program(${QT_QMAKE_EXECUTABLE} ARGS "-v" OUTPUT_VARIABLE QT_QMAKE_VERSION_INFO)
-exec_program(${QT_QMAKE_EXECUTABLE} ARGS "-query QT_INSTALL_LIBS" OUTPUT_VARIABLE QT_INSTALL_LIBS)
-exec_program(${QT_QMAKE_EXECUTABLE} ARGS "-query QT_INSTALL_BINS" OUTPUT_VARIABLE QT_INSTALL_BINS)
-
+execute_process(COMMAND ${QT_QMAKE_EXECUTABLE} -v OUTPUT_VARIABLE QT_QMAKE_VERSION_INFO OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
I actually think everything related to Qt is not needed anymore/should not be needed for pyopenms.
It is only used in the unused setup.py build step.
If we cleaned up the setup.py or replace it with a project.toml, we would probably see this better.

If we just want the version, to display it in init.py to warn users about other pyqt installations, we could get it from the cmake find module.